docs/file-formats/json-project/ontologies.md

```diff
@@ -1072,69 +1072,3 @@ Example for a resource definition:
   ]
 }
 ```
-
```

Comment:
These 2 sections already live in caveats.md. I forgot to remove them after copying them to caveats.md